moderatorguide.md

```diff
@@ -36,7 +38,7 @@ Unless otherwise agreed by the organizers, the following guidelines apply:
 * Short talks: after the talk ends.
 * Long talks: questions may be asked at any time.
```

Comment:
Is this still the case? I thought we discussed that every organizer will decide on their own way to ask questions.

```suggestion
As a general rule this is the format for:
* Short talks: after the talk ends.
* Long talks: questions may be asked at any time.
```

Reply:
I think it is still good to mention a suggested format, but we can use softer language, like "Our suggested guidelines for when questions are allowed:"

Reply:
```suggestion
Our suggested guidelines for when questions are allowed:
* Short talks: after the talk ends
* Long talks: questions may be asked at any time
```